dataland-frontend/tests/e2e/specs/low/pages/taxonomy/IndexPanelTabMenu.ts

```diff
@@ -29,24 +38,15 @@ describe('Index Panel behavior', function () {
     });
 
 
+    it('Visit searchtaxonomy page, scroll to the bottom, back to the top, and check if Dax still highlighted', () => {
+        cy.restoreLoginSession()
+        cy.visit('/searchtaxonomy')
 
-    it('Index tabmenu should be present', () => {
-        cy.visit("/searchtaxonomy")
-        cy.get(indexTabMenu).should('exist')
-        cy.get('.grid')
-            .should('not.contain','Choose by stock market index')
-        cy.get('h2')
-            .should('contain', "Results")
-        cy.get('table.p-datatable-table').should('exist')
-        cy.get('table.p-datatable-table').contains('th','COMPANY')
-        cy.get('table.p-datatable-table').contains('th','SECTOR')
-        cy.get('table.p-datatable-table').contains('th','MARKET CAP')
-        cy.get('table.p-datatable-table').contains('td','VIEW')
-            .contains('a', 'VIEW')
-            .click()
-            .url().should('include', '/companies/')
-            .url().should('include', '/eutaxonomies')
-    });
+        checkIfDaxTabIsHighlighted()
```

Comment:
Vielleicht sollten wir in Hinblick auf noch andere Testblöcke, einen neuen Task erstellen, der das noch weiter modular aufbaut. Dieser ganze Block mit den p-datatables wird auch in dem Test von searchtaxonomy verwendet.

Reply:
Ich würde denken dass wir das mit dem Zielbild-Termin den wir uns vorgenommen mit erschlagen. 
Kannst du es da mitnehmen?

Reply:
Ich verstehe nicht, weshalb der Kommentar an dieser Code-Stelle steht.
Hier finden keine Tests auf p-datatables statt.
Und das `checkIfDaxTabIsHighlighted()` ist ein exklusiver Check für diesen Test, weshalb er nur hier definiert und genutzt wird.